### PR TITLE
Add user-agent header

### PR DESCRIPTION
**Pull request by timoinstana**
# Why / What
Add user-agent header Instana/Concourse to be able to track which sources create release markers.

### CODE REVIEW

File: assets/out [hunk 1]
```diff
@@ -86,7 +86,7 @@ fi
 
 echo "Using the API endpoint: ${endpoint}/api/releases"
 
-if ! release=$(curl --silent --show-error --fail -X POST --header 'Content-Type: application/json' --header "Authorization: apiToken ${api_token}" "${endpoint}/api/releases" -d "{
+if ! release=$(curl --silent --show-error --fail -X POST --header 'Content-Type: application/json' --header "Authorization: apiToken ${api_token}" --header 'User-Agent: Instana/Concourse' "${endpoint}/api/releases" -d "{
```
Comment (mmanciop) [inline assets/out]: 'instana/pipeline-feedback-resource'

And ideally we'd add the version there too

**Review by mmanciop — CHANGES_REQUESTED** [review]

File: assets/out (hunk 1 above)
Comment (timoinstana) [reply assets/out]: Good idea to track the version here as well

File: assets/check [hunk 2]
```diff
@@ -70,5 +75,5 @@ fi
 echo 'Release not found'
 echo 'Fetching the latest release instead'
 
-curl --silent --show-error --fail --header "Authorization: apiToken ${api_token}" "${endpoint}/api/releases" \
+curl --silent --show-error --fail --user-agent "instana/pipeline-feedback-resource/${version}" --header "Authorization: apiToken ${api_token}" "${endpoint}/api/releases" \
```
Comment (timoinstana) [inline assets/check:78]: **[question]** All other places have the `v` prefix for versions. Why not here?

Comment (mmanciop) [reply assets/check:78]: I removed the v in too few places

Comment (mmanciop) [reply assets/check:78]: Addressed in #242e75cacd00ef61e3e2acc4fa08825337dc4ad3

File: assets/out (hunk 1 above)
Comment (mmanciop) [reply assets/out]: Addressed in eb39436

**Review by mmanciop — APPROVED** [review]

**Pull request merged**